packages/compass-components/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I like what's already here a lot, and would consider adding some more recommended options from typescript docs:

- [forceConsistentCasingInFileNames](https://www.typescriptlang.org/tsconfig#forceConsistentCasingInFileNames)
- [exactOptionalPropertyTypes](https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes)
- [skipLibCheck](https://www.typescriptlang.org/tsconfig#skipLibCheck)